animation_nodes/sockets/midi_note.py

```diff
@@ -0,0 +1,42 @@
+import bpy
+from .. base_types import AnimationNodeSocket, PythonListSocket
+
+class MIDINoteSocket(bpy.types.NodeSocket, AnimationNodeSocket):
+    bl_idname = "an_MIDINoteSocket"
+    bl_label = "MIDI Note Socket"
+    dataType = "midiNote"
+    drawColor = (0.9, 0.7, 0.4, 1)
+    storable = True
+    comparable = False
+
+    def getCurrentDataType(self):
```

Comment:
I don't think this should be here.

Reply:
I take an example from another socket. I don't know what it's for
What do you think about the values used for drawColor ? Are these the right colors I've used ?

Reply:
I haven't checked the colors yet. But lets leave those details to the end.